src/pudl/metadata/codes.py

```diff
@@ -1115,4 +1130,33 @@
         "code_fixes": {},
         "ignored_codes": [],
     },
+    "steam_plant_type_eia": {
+        "df": pd.DataFrame(
+            columns=["code", "label", "description"],
+            data=[
+                (
+                    1,
+                    "steam over 100MW",
```

Comment:
I have been requiring all of the `label` columns using `snake_case`

Reply:
ah got it! is that communicated anywhere?

Reply:
i see it in the module docs!